build.gradle

```diff
@@ -105,11 +105,21 @@ task clean(type: Delete) {
     delete rootProject.buildDir
 }
 
+def launchTask = getGradle()
```

Comment:
for some extra context, querying the start task command like this allows us to avoid including extra scripts/plugins into the main script until we need them